scripts/content-repo-preview/build.ts

```diff
@@ -0,0 +1,99 @@
+import { promisify } from 'util'
+import { execFileSync } from 'child_process'
+import path from 'path'
+import fs from 'fs'
+
+async function main() {
+  // imports
+  const execFile = promisify((await import('child_process')).execFile)
```

Comment:
❔ What's the thought behind `execFile` vs `execFileSync` here?

Reply:
🤷‍♂️  I had initially started with this, but I realized I needed stdio to come through in the parent process, so pivoted to `execFileSync`, which promisified `execFile` doesn't support as neatly. I'll probably switch everything to just use `execFileSync`.